spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -239,6 +239,16 @@ def price_tier
         client.price_tier(self.apple_id)
       end
 
+      # set the availability. This method doesn't require `save` to be called
+      def update_availability!(territories, opts = {})
```

Comment:
In this implementation, `update_availability` does not take the same type of object as what is returned by `availability`. I.e., `update_availability` takes an array of territories, while `availability` returns an `Availability` object.

So, you can't do something like this without an error:
```
availability = app.availability
app.update_availability!(availability)
```
I could see this as a source of confusion for users.

Some ideas:
- Make `update_availability` take an `Availability` object and create a constructor for `Availability` that takes an array of territories and an optional `include_future_territories` value.
- Split the `availability` accessors into separate `availability_territories` and `availability_include_future_territories` accessors

Thoughts?